src/Hosting/Hosting/src/Internal/HostingApplication.cs

```diff
@@ -78,6 +78,7 @@ public Context CreateContext(IFeatureCollection contextFeatures)
             }
 
             _diagnostics.BeginRequest(httpContext, hostContext);
+            httpContext.Features.Set<IHttpActivityFeature>(new ActivityFeature());
```

Comment:
Add to Kestrel as a known feature?
https://github.com/dotnet/aspnetcore/blob/698d872e378c6241432c97e65bacf3bc0af1f4d0/src/Servers/Kestrel/tools/CodeGenerator/HttpProtocolFeatureCollection.cs#L8

Reply:
I haven't had the chance to fix it yet. I just pushed a rename before API review today so that we don't bikeshed on the name

Reply:
Take a look and see if I did this correctly